packages/account/src/Sections/Profile/PhoneVerification/cancel-phone-verification-modal.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button, Text } from '@deriv-com/quill-ui';
+import { Modal } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { useHistory } from 'react-router';
+
+type TCancelPhoneVerificationModal = {
+    should_show_cancel_verification_modal: boolean;
+    setShouldShowCancelVerificationModal: (value: boolean) => void;
+};
+
+const CancelPhoneVerificationModal = ({
+    should_show_cancel_verification_modal,
+    setShouldShowCancelVerificationModal,
+}: TCancelPhoneVerificationModal) => {
+    const history = useHistory();
+    const handleCancelButton = () => {
+        setShouldShowCancelVerificationModal(false);
+        history.goBack();
+    };
+
+    return (
+        <Modal className='phone-verification__cancel-modal' is_open={should_show_cancel_verification_modal}>
+            <Modal.Body>
+                <div className='phone-verification__cancel-modal--contents'>
+                    <Text bold>
+                        <Localize i18n_default_text='Cancel phone number verification?' />
+                    </Text>
+                    <Text>
+                        <Localize i18n_default_text='All details entered will be lost.' />
+                    </Text>
+                </div>
+            </Modal.Body>
+            <Modal.Footer>
+                <div className='phone-verification__cancel-modal--buttons'>
+                    <Button
```

Comment:
can we use FormSubmitButton component

Reply:
I thought we are using button components from quill-ui 🤔

Reply:
oh yea... Forgot it
Thanks for checking @suisin-deriv